local-dev: add pgadmin4 container

jan-zmeskal opened the pull request:
The purpose here is to make clair-db introspection easier. With this change, a developer can use [pgadmin](https://www.pgadmin.org/) browser GUI to view clair-db database. Having it run in container and pre-configured removes the need for a developer to install it manually on their system and set up clair-db connection every time. With this change, one can simply:
1. `make local-dev-up`
1. Enter `localhost:80` in the browser
1. Log-in with the default user (clair@clair.com/clair)
1. Inspect the clair-db (no need for setting up the connection, it's already pre-loaded)
![image](https://user-images.githubusercontent.com/22600243/95863188-2d219b80-0d64-11eb-8c24-5a481713c248.png)

I don't know about exposing port 80 though. It's the one that is used in [pdagmin4 docs](https://www.pgadmin.org/docs/pgadmin4/development/container_deployment.html), so I chose it for compliance. At the same time, I know it's heavily contested port. What do you think?

Commit:
local-dev: add pgadmin4 container

Signed-off-by: Jan Zmeskal <jzmeskal@redhat.com>

ldelossa commented on docker-compose.yaml:
```diff
@@ -24,6 +24,20 @@ services:
       start_period: 10s
 
+  pgadmin:
+    container_name: pgadmin
```
Safe to pin this to "latest" ? Or should we be more strict?

jan-zmeskal replied:
Hard to say. I don't have long-term experience with this image as previously I had pgadmin just installed on my system. As this is a development tool and read-only at that, I don't think it's an issue and I think we can go with latest.

ldelossa commented on docker-compose.yaml:
```diff
@@ -24,6 +24,20 @@ services:
+      PGADMIN_DEFAULT_PASSWORD: clair
+      PGADMIN_SERVER_JSON_FILE: /pgadmin4/config/servers.json
+    ports:
+      - "80:80"
```
Can we move this to a less common port? Taking port 80 seems a bit brazen and requires sudo.

jan-zmeskal replied:
Yeah, I also think so. What do you think would be a good port number, @ldelossa ? I was thinking maybe 8081 since we already use 8080 and 8082.

ldelossa reviewed (changes requested):
Nice add. Few comments in the pull but this seems like a good idea.

jan-zmeskal commented:
@ldelossa I updated pgadmin to use port 8081. Also please note that I did two additional things:
1. I noticed the command to restart rabbitmq in `Makefile` was wrong so I've fixed it 
2. I added command `local-dev-clair-restart` to restart all the instances of clair (indexer, matcher and notifier)

ldelossa reviewed (dismissed):
LGTM.

ldelossa commented:
@jan-zmeskal thanks for the adds. Looks good. Needs a rebase however.

jan-zmeskal commented:
@ldelossa Can you pls approve again? Sorry for that, I needed to rebase
